interfaces/cython/cantera/test/test_convert.py

```diff
@@ -478,26 +470,24 @@ def test_extra(self):
                      transport='gri30_tran.dat', output='gri30_extra',
                      extra='extra.yaml')
 
-        output = pjoin(self.test_work_dir, 'gri30_extra' + self.ext)
-        with open(output, 'rt', encoding="utf-8") as stream:
-            yml = yaml.safe_load(stream)
+        output = self.test_work_path / ("gri30_extra" + self.ext)
+        yml = utilities.load_yaml(output)
 
         desc = yml['description'].split('\n')[-1]
         self.assertEqual(desc, 'This is an alternative description.')
         for key in ['foo', 'bar']:
             self.assertIn(key, yml.keys())
         # This test tests it can convert the SRI parameters when D or E equal to 0
```

Comment:
I think this comment is too far indented, and I think it should be a docstring rather than a comment.

Reply:
Looks like it is the docstring for the next test - this was not changed, but I think it's worth fixing.